libs/mini-rx-store/src/lib/component-store.ts

```diff
@@ -80,6 +67,8 @@ export class ComponentStore<StateType extends object>
             }
         });
 
+        const metaReducers: MetaReducer<StateType>[] = [];
```

Comment:
Meta Reducers is not yet correct.

See in Signal Store lib:

![image](https://github.com/spierala/mini-rx-store/assets/1272446/ae7a9f35-a3bd-4b2a-8cb2-0b46f3dd57e7)

Reply:
same for reducer.

Maybe it would be cool to have a `calculateComponentStoreReducer` in the common lib, which uses calculateExtensions under the hood?

Then we have less possibilities to introduce bugs here :)

Can be done in another PR if we want that.

Reply:
The issue in refactoring that is that `ComponentStore` in `mini-rx-store` can be initialized with undefined.

```
export class ComponentStore<StateType extends object> {
...
    constructor(initialState?: StateType, config?: ComponentStoreConfig) {
        ...
    }
}
```

While `ComponentStore` in `signal-store` can not:

```
constructor(private initialState: StateType, config?: ComponentStoreConfig) {
    ...
}
```

This is an issue when then calling `combinedMetaReducer` as Typescript detects incompatible types. So it's gonna be a bit more of a refactor than what you maybe had in mind?

Reply:
OK, I see.

reducer is calculated later with an initialState in mini-rx-store.

OK that refactor is then indeed less straight-forward than expected.

Still here the metaReducers have to be calculated by doing this:

`const metaReducers: MetaReducer<StateType>[] = extensions.map((ext) => ext.init());`